build/webbundle.xf.js

```diff
@@ -21,6 +21,7 @@ exports.make = async function(appname, webbundlepath, filePatterns) {
     try {
         if ((!appname) || appname.toLowerCase() == "use_default_app") appname = DEFAULT_APP_NAME;
         if (!webbundlepath) webbundlepath = `${MONKSHU_PATH}/frontend/apps/${appname}/webbundle/webbundle.json`;
+        else webbundlepath = webbundlepath + "/webbundle/webbundle.json";
```

Comment:
If `webbundlepath` is provided then it is the complete path to the bundle JSON file!